docs/samples/browser-plugin-meetings/index.html

```diff
@@ -363,7 +363,10 @@ <h2 class="collapsible">Meeting Management</h2>
                     <input type="checkbox" name="ts-media-direction" value="videoEnabled" checked="">videoEnabled
                   </label>
                   <label>
-                    <input type="checkbox" name="ts-media-direction" value="receiveShare" checked="">receiveShare
+                    <input type="checkbox" name="ts-media-direction" value="enableShareAudio" checked="">enableShareAudio
```

Comment:
This doesn't look right. the value is used in app.js as the name of the property that's then passed into calls to addMedia() and updateMedia(). So here the name is `enableShareAudio` and it matches what addMedia() takes, but updateMedia takes `shareAudioEnabled`. 
I think we should rename the prop that addMedia() takes to `shareAudioEnabled`, because that will fit more with the other props from addMedia like `audioEnabled` and `videoEnabled` and then also rename it here to `shareAudioEnabled`.
Same applies to share video.

Reply:
My bad. I thought they both were `enableShareAudio`

Will update and raise a PR. Thanks for noticing it @marcin-bazyl

Reply:
Done @marcin-bazyl